.github/workflows/generate-docs.yml

```
@@ -0,0 +1,147 @@
+name: Generate documentation
+on:
+  push:
+    branches:
+      - main
+      - release*
+    tags:
+      - '[0-9]+.[0-9]+*'
+
+env:
+  PACKAGE_NAME: numba-dppy
+  python: 3.8
+  artifact_name: -c dppy_label_dev
+  dependencies: dpnp=0.7.1=*_41
+  CHANNELS: -c dppy/label/dev -c defaults -c numba -c intel -c numba/label/dev -c conda-forge --override-channels
```

Remove `-c dppy/label/dev` from highest priority.
```suggestion
  CHANNELS: -c defaults -c numba -c intel -c numba/label/dev -c dppy/label/dev -c conda-forge --override-channels
```